src/pages/BlogOverviewPage.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import posts from '../data/posts.json';
+import {NavLink} from "react-router-dom";
+
+function BlogOverviewPage() {
+
+    const blogLink = posts.map((posts) => {
+        return <li> <NavLink to={`/blog/${posts.id}`}> {posts.title} </NavLink> </li>
```

Comment:
dit hoeft geen Navlink te zijn, link is voldoende

Reply:
Ik merk dat jij je code ook meer op 1 regel laat staan,
ikzelf doe dat ook, omdat ik het (zeker voor hele korte code) leesbaarder vind.